setup.py

```diff
@@ -57,6 +58,12 @@ def get_new_url(url):
 with open("rig/version.py", "r") as f:
     exec(f.read())
 
+# The requirements vary depending on the version of Python
+requirements = ["numpy>1.6", "six", "sentinel", "pytz"]
+if sys.version_info[0] < 3 or sys.version_info[1] < 4:
```

Comment:
When you compare tuples it does element wise comparison from left-to-right.
Kind'a slick!

On 2 October 2015 at 13:01, Andrew Mundy <notifications@github.com> wrote:

> In setup.py
> <https://github.com/project-rig/rig/pull/186#discussion_r41014386>:
>
> > @@ -57,6 +58,12 @@ def get_new_url(url):
> >  with open("rig/version.py", "r") as f:
> >      exec(f.read())
> >
> > +# The requirements vary depending on the version of Python
> > +requirements = ["numpy>1.6", "six", "sentinel", "pytz"]
> > +if sys.version_info[0] < 3 or sys.version_info[1] < 4:
>
> I didn't realise one could do that...
>
> (Not that I'm entirely certain I know how that gets parsed)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/project-rig/rig/pull/186/files#r41014386>.
>